packages/neherlab-covid-19-scenarios/src/basel-api.ts

```diff
@@ -89,61 +144,76 @@ export interface ExposedCurrentData {
   overflow: Record<string, number>
 }
 
-export interface CumulativeData {
+export interface ExposedCumulativeData {
   recovered: Record<string, number>
   hospitalized: Record<string, number>
   critical: Record<string, number>
   fatality: Record<string, number>
 }
 
+export interface TimePoint {
+  t: number
```

Comment:
So, this is indexed by number of days since t<sub>0</sub>? If that's right, you could do:

`type DaysFromStart = number;`

And then:

`t: DaysFromStart`

(Or some other name) and replace `number` with `DaysFromStart` wherever it makes sense. Then this all becomes clear and self-documenting.

Reply:
You are right, but for now I am copying these verbatim from the model's generated types for consistency. In future I hope we can get rid of most of these and consume/realias their types from NPM.